tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -199,12 +200,18 @@ function clusterRebalanceOtherOptionsSuite() {
       }
     },
 
-    tearDownAll: function () {
+    tearDownAll: function() {
       db._useDatabase(prevDB);
       db._dropDatabase(database);
     },
 
-    testCalcRebalanceStopServer: function () {
+    testRebalancePlanWithoutSystemCollections: function() {
+      const result = getRebalancePlan(true, true, true, true);
+      assertEqual(result.result.imbalanceBefore.shards.totalShardsFromSystemCollections, 0);
+      assertEqual(result.result.imbalanceAfter.shards.totalShardsFromSystemCollections, 0);
```

Comment:
Ok, so here we verify that there are no shards from system collections in the plans.
But we don't verify that we *have* system collection shards when system collections are *not* excluded.
We should add such assertions.